### PR TITLE
 Rename nxphlite-v3 to NXP fmuk66-v3

### PR DESCRIPTION
**Pull request by davids5**
FYI @igalloway This PR Renames nxphlite-v3 to NXP fmuk66-v3

### CODE REVIEW

**Comment by dagar** [comment]
We'll need to update QGC as well.

File: boards/nxp/fmuk66-v3/nuttx-config/nsh/defconfig [hunk 1]
```diff
@@ -809,7 +809,7 @@ CONFIG_CDCACM_TXBUFSIZE=8000
 CONFIG_CDCACM_VENDORID=0x1FC9
 CONFIG_CDCACM_PRODUCTID=0x001c
 CONFIG_CDCACM_VENDORSTR="NXP SEMICONDUCTORS"
-CONFIG_CDCACM_PRODUCTSTR="PX4 NXPHlite v3.x"
+CONFIG_CDCACM_PRODUCTSTR="PX4 FMUK66 v3.x"
```
Comment (dagar) [inline boards/nxp/fmuk66-v3/nuttx-config/nsh/defconfig]: ?

```suggestion
CONFIG_CDCACM_PRODUCTSTR="NXP FMUK66 v3.x"
```

Comment (mhkabir) [reply boards/nxp/fmuk66-v3/nuttx-config/nsh/defconfig]: Shouldn't the product string not contain the vendor at all? Since there is a VENDORSTR.

Comment (davids5) [reply boards/nxp/fmuk66-v3/nuttx-config/nsh/defconfig]: I wonder....

One way to look at it is: It is what the Product is: A PX4 fmu on a K66  

Bootloaders have "PX4 BL NXPHlite v3.x" and the wild card is done on the "PX4 BL*" and the rest i

So the Bootloader will have to be done too.  (or not. @igalloway this is up to you)

**Comment by dagar** [comment]
Let's merge now and continue discussing the VENDORSTR in an issue (if desired).

**Pull request merged**